src/components/ButtonDropdown/ButtonDropdown.tsx

```diff
@@ -180,9 +171,7 @@ export const ButtonDropdown = ({
       {dropdownMenuTriggerWithProps}
       <DropdownMenu
         className={styles['button-dropdown__dropdown-menu']}
-        handleOnClick={closePanel}
-        handleOnEscDown={(e) => handleKeyDown(e)}
-        handleOnTabDown={(e) => handleKeyDown(e)}
+        closeDropdownMenu={closePanel}
```

Comment:
No longer closes on click, (should be brought back once project card uses popover).